src/SentryExporter.php

```diff
@@ -31,9 +31,10 @@ class SentryExporter
      * SentryExporter constructor.
      *
      * @param string $token
+     * @param string $useThrottling
      * @param string $sentryBase
      */
-    public function __construct(string $token, string $sentryBase = 'https://sentry.io/api/0/')
+    public function __construct(string $token,string $useThrottling, string $sentryBase = 'https://sentry.io/api/0/')
```

Comment:
Introducing a new mandatory parameter would make this a breaking change. Although I see this as a key feature for large accounts, it would be nice to not make this breaking.
So I would suggest to add this as optional third parameter instead and make it `false` by default. So anyone using just the PHP class instead of the docker image will not be forced to adjust code because of this new feature.